### PR TITLE
docs(module:table): document sort properties

### PR DESCRIPTION
**Pull request by krokofant**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[X] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: #4610 

## Does this PR introduce a breaking change?
```
[ ] Yes
[X] No
```

## Other information
I've only added translations for en-US and the zh-CN needs to be added. @vthinkxie

### CODE REVIEW

File: components/table/doc/index.en-US.md [hunk 1]
```diff
@@ -127,6 +127,17 @@ Filter property
...
 
+Sort property
+
+| Property | Description | Type | Default |
+| -------- | ----------- | ---- | ------- |
+| `[nzShowSort]` | Whether to show sort | `boolean` | - |
+| `[nzSort]` | Sort direction | `'ascend' | 'descend' | null` | - |
+| `[nzSortKey]` | Key to sort by | `string` | - |
+| `(nzSortChange)` | Sort direction change callback | `EventEmitter<'ascend' \| 'descend' \| null>` | - |
+| `(nzSortChangeWithKey)` | Sort direction change callback with `key` | `EventEmitter<{key: string, value: 'ascend' \| 'descend' \| null}>` | - |
```
Comment (hsuanxyz) [inline components/table/doc/index.en-US.md]:
```suggestion
```

@vthinkxie Looks like a private API, need to make it public in the docs?

Comment (krokofant) [reply components/table/doc/index.en-US.md]: What makes the `nzSortChangeWithKey` a private api and not the others with a similar signature?

```typescript
  @Output() readonly nzCheckedChange = new EventEmitter<boolean>();
  @Output() readonly nzSortChange = new EventEmitter<string | null>();
  @Output() readonly nzSortChangeWithKey = new EventEmitter<{ key: string; value: string | null }>();
  /* tslint:disable-next-line:no-any */
  @Output() readonly nzFilterChange = new EventEmitter<any[] | any>();
```

Comment (hsuanxyz) [reply components/table/doc/index.en-US.md]: 🤔We should rename it to `sortChangeWithKey`.

File: components/table/doc/index.en-US.md [hunk 2]
```diff
@@ -135,7 +135,6 @@ Sort property
 | `[nzSort]` | Sort direction | `'ascend' | 'descend' | null` | - |
 | `[nzSortKey]` | Key to sort by | `string` | - |
 | `(nzSortChange)` | Sort direction change callback | `EventEmitter<'ascend' \| 'descend' \| null>` | - |
-| `(nzSortChangeWithKey)` | Sort direction change callback with `key` | `EventEmitter<{key: string, value: 'ascend' \| 'descend' \| null}>` | - |
```
Comment (hsuanxyz) [inline components/table/doc/index.en-US.md]: Oops, Apply this suggestion caused the [Semantic Pull Request](https://github.com/zeke/semantic-pull-requests) check is fail.

Please change this commit message(`"Update components/table/doc/index.en-US.md"`), or rebase it. Thank you very much.

**Pull request merged**